packages/terra-application/tests/wdio/application-container/skip-to-links-spec.js

```diff
@@ -2,7 +2,7 @@ const selector = '#root';
 
 Terra.describeViewports('ApplicationContainer SkipToLinks', ['large'], () => {
   before(() => {
-    browser.url('#/skip-to-links');
+    browser.url('terra-application-test/#/skip-to-links');
```

Comment:
the url is now scoped because the terra-application test harness runs within the same server as terra-dev-site